introspection-engine/introspection-engine-tests/tests/identify_version/mod.rs

```diff
@@ -215,7 +215,7 @@ async fn introspect_postgres_prisma2(api: &TestApi) -> TestResult {
 
 //Mysql
 
-#[test_connector(tags(Mysql))]
+#[test_connector(tags(Mysql), exclude(TiDB))]
 async fn introspect_mysql_non_prisma(api: &TestApi) -> TestResult {
```

Comment:
What is the reasoning behind this test @do4gr - Do we have a "This has an unknown type, this can not be Prisma 1" in our `NonPrisma` detection logic?